src/stateMachines/maintenance/maintenance.ts

```diff
@@ -49,7 +52,8 @@ export class Maintenance {
           CHANGE_PASSWORD: { target: 'CHANGE_PASSWORD' },
           SYNC_TIME: { target: 'SYNC_TIME' },
           SYNC_IP: { target: 'SYNC_IP' },
-          SYNC_HOST_NAME: { target: 'SYNCH_HOST_NAME' }
+          SYNC_HOST_NAME: { target: 'SYNCH_HOST_NAME' },
+          SYNC_DEVICE_INFO: { target: 'SYNCH_DEVICE_INFO' }
```

Comment:
Can we match the SYNC_TIME, and SYNC_IP? meaning these should be:
``` typescript
SYNC_HOST_NAME: { target: 'SYNC_HOST_NAME' },
SYNC_DEVICE_INFO: { target: 'SYNC_DEVICE_INFO' }
```

Reply:
fixed